includes/class-telemetry.php

```
@@ -0,0 +1,541 @@
+				curl_setopt( $ch, CURLOPT_HEADER, true );
+				curl_setopt( $ch, CURLOPT_FOLLOWLOCATION, true );
+				curl_setopt( $ch, CURLOPT_ENCODING, '' ); // Auto-decode gzip/brotli/deflate.
+				curl_setopt( $ch, CURLOPT_MAXREDIRS, 5 );
+				curl_setopt( $ch, CURLOPT_TIMEOUT, 30 );
+				curl_setopt(
+					$ch,
+					CURLOPT_USERAGENT,
+					'WordPress/' . get_bloginfo( 'version' ) . '; ' . home_url()
+				);
+				curl_setopt( $ch, CURLOPT_SSL_VERIFYPEER, false );
+				curl_setopt( $ch, CURLOPT_SSL_VERIFYHOST, false );
```

PHPStan (2.1.46)

[warning] Parameter `#3` $value of function curl_setopt expects 0|2, false given.


(argument.type)